go.mod

```diff
@@ -12,5 +12,5 @@ require (
 	github.com/nicksnyder/go-i18n/v2 v2.0.3
 	github.com/pkg/errors v0.9.1
 	github.com/stretchr/testify v1.6.1
-	golang.org/x/sys v0.2.0 // indirect
+	golang.org/x/sys v0.0.0-20220520151302-bc2c85ada10a // indirect
```

Comment:
What was the need of changing this version?

Reply:
Its their comments was to use the same version as the master so I copied this from there.